vendor/engines/badges_engine/spec/spec_helper.rb

```diff
@@ -0,0 +1,30 @@
+require 'simplecov'
+ENV["RAILS_ENV"] ||= 'test'
+require File.expand_path("../dummy/config/environment.rb", __FILE__)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.